docs/api/redirect.md

```diff
@@ -6,14 +6,13 @@ sidebar_position: 5
 toc_max_heading_level: 5
 ---
 
-Is used to redirect the ctx(request) to a different URL/Route.
+The redirect package is used to redirect the context (request) to a different URL or route.
```

Comment:
I think package is not convenient here. Maybe we can replace it to the Redirect methods?

Reply:
Yeah, this is a bug from ChatGPT 😂 it shouldn't say package